packages/react-search-ui-views/src/styles/themes/reference-ui/components/_paging.scss

```diff
@@ -18,14 +31,6 @@
       color: $linkColor;
```

Comment:
New version of library using same color on pagination-item on hover event so it seemed like hover doesn't work at all